python/ray/serve/tests/test_config_files/pid.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, dag: RayServeDAGHandle):
         self.dag = dag
 
     async def __call__(self):
-        return await self.dag.remote()
+        return await (await self.dag.remote())
```

Comment:
I'm pretty sure this code didn't work before 🤦‍♀️. Not sure how all the tests were passing in `test_deploy_config_update`, but some of them were because I wasn't casting the response to int, so I've updated the test to do that.